examples/util/tflite/TFliteModelImporter.js

```diff
@@ -38,9 +38,13 @@ class TFliteModelImporter {
     this._execution = await this._compilation.createExecution();
   }
 
-  async compute(inputTensor, outputTensor) {
-    this._execution.setInput(0, inputTensor);
-    this._execution.setOutput(0, outputTensor);
+  async compute(inputTensors, outputTensors) {
```

Comment:
Here support the mutiple inputs/outputs. And all the examples that use TFliteImporter also do some changes for it.